marconi-chain-index/test-lib/Helpers.hs

```diff
@@ -37,58 +39,28 @@ the near future we're planning to replace cardano-testnet with cardano-node-emul
 cardano-testnet related code should change anyway.
 -}
 
-{- | Start a testnet.
- startTestnet
-   :: TN.TestnetOptions
-   -> FilePath
-   -> FilePath
-   -> H.Integration (C.LocalNodeConnectInfo C.CardanoMode, TC.Conf, TN.TestnetRuntime)
- startTestnet testnetOptions base tempAbsBasePath' = do
-   configurationTemplate <- H.noteShow $ base </> "configuration/defaults/byron-mainnet/configuration.yaml"
-   conf :: TC.Conf <-
-     HE.noteShowM $
-       TC.mkConf
-         (TC.ProjectBase base)
-         (TC.YamlFilePath configurationTemplate)
-         (tempAbsBasePath' <> "/")
-         Nothing
-   tn <- TN.testnet testnetOptions conf
--}
-
---   -- Boilerplate codecs used for protocol serialisation.  The number
---   -- of epochSlots is specific to each blockchain instance. This value
---   -- what the cardano main and testnet uses. Only applies to the Byron
---   -- era.
---   socketPathAbs <- getPoolSocketPathAbs conf tn
---   let epochSlots = C.EpochSlots 21600
---       localNodeConnectInfo =
---         C.LocalNodeConnectInfo
---           { C.localConsensusModeParams = C.CardanoModeParams epochSlots
---           , C.localNodeNetworkId = getNetworkId tn
---           , C.localNodeSocketPath = socketPathAbs
---           }
-
---   pure (localNodeConnectInfo, conf, tn)
-
--- getNetworkId :: TN.TestnetRuntime -> C.NetworkId
--- getNetworkId tn = C.Testnet $ C.NetworkMagic $ fromIntegral (TN.testnetMagic tn)
-
--- getSocketPathAbs :: (MonadTest m, MonadIO m) => TC.Conf -> TN.TestnetRuntime -> m FilePath
--- getSocketPathAbs conf tn = do
---   let tempAbsPath = TC.tempAbsPath conf
---   socketPath <- IO.sprocketArgumentName <$> H.headM (TN.nodeSprocket <$> TN.bftNodes tn)
---   H.note =<< (liftIO $ IO.canonicalizePath $ tempAbsPath </> socketPath)
-
--- getPoolSocketPathAbs :: (MonadTest m, MonadIO m) => TC.Conf -> TN.TestnetRuntime -> m FilePath
--- getPoolSocketPathAbs conf tn = do
---   let tempAbsPath = TC.tempAbsPath conf
---   socketPath <- IO.sprocketArgumentName <$> H.headM (TN.poolSprockets tn)
---   H.note =<< (liftIO $ IO.canonicalizePath $ tempAbsPath </> socketPath)
+-- | Start a testnet.
+startTestnet
+  :: FilePath
+  -> H.Integration (C.LocalNodeConnectInfo C.CardanoMode, C.NetworkId, FilePath)
+startTestnet tempAbsBasePath = do
+  let socketPathAbs = tempAbsBasePath <> "/node-server.sock"
+      networkId = testnet -- TODO: any other networkId doesn't work
+      epochSlots :: (Num a) => a
```

Comment:
I wonder if it would be worth changing `E.startTestnet` to take a `Word64` instead of an `Integer`